data/human/free worlds side plots.txt

```diff
@@ -808,3 +808,74 @@ mission "FW Stack Core 1C"
 		conversation
 			`You receive a message from Freya: "Hello again, Captain! Thanks for your help escorting the materials to Solace. Ever since we developed the Dreadnoughts people have been talking about how badly we need a generator that can meet their energy needs, and that's what Delta V Corporation has been working on. So, take a look next time you're on Solace or one of our other major manufacturing centers. Sorry for the mystery surrounding the cargo, but we didn't exactly want to advertise that a huge shipment of fissionable material was headed for Free Worlds space."`
 				decline
+
+
+
+mission "FW Refugees to Humanika"
+	name "Transport Refugee Family"
+	description "Transport a family of <bunks> refugees and <cargo> to <destination>. Payment is <payment>."
+	minor
+	source
+		attributes "dirt belt" rim south
+		not government Quarg
+		not near Tarazed 1
+	destination Humanika
+	to offer
+		has "event: war begins"
+		not "chosen sides"
+		random < 15
+	to fail
+		has "event: joined the free worlds"
+	passengers 3
+	cargo "household goods" 2
+	on offer
+		conversation
+			`A young family of three is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`
+			choice
+				`	(Approach them.)`
+				`	(Ignore them.)`
+					defer
+			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`
```

Comment:
> You should clarify which one of them is speaking to the player.

Does it matter? It seemed irrelevant to me, so I didn't include it.

> `Quarg planet of <destination>`

Isn't either way correct?

Reply:
There isn't anything we gain by not specifying the speaker.

"The Quarg planet of Humanika" is more aesthetically pleasing. Compare "the American city of Chicago" with "the American city Chicago".

Reply:
> There isn't anything we gain by not specifying the speaker.

Or by specifying the speaker either, so better to not give information that doesn't add anything, right? That way if there's a reason it should be one speaker or the other in the future, it can be specified then without changing anything that already happened.

> "The Quarg planet of Humanika" is more aesthetically pleasing. Compare "the American city of Chicago" with "the American city Chicago".

I don't think I would ever use "of" in "the American city of Chicago" either. (It's correct, just not something I'd say/write.) Is there a relevant style guide to resolve this?